src/main/kotlin/no/nav/syfo/person/api/PersonAPI.kt

```diff
@@ -154,17 +157,19 @@ fun Route.registrerPersonApi(
 
                 val callId = getCallId()
 
-                val person = pdlClient.person(
+                val pdlPerson = pdlClient.person(
                     callId = callId,
                     personIdentNumber = personIdentNumber,
                 )
-                val response = PersonAdresseResponse(
-                    navn = person?.getFullName() ?: "",
-                    bostedsadresse = person?.bostedsadresse(),
-                    kontaktadresse = person?.kontaktadresse(),
-                    oppholdsadresse = person?.oppholdsadresse()
-                )
-                call.respond(response)
+                pdlPerson?.hentPerson?.let { person ->
+                    val response = PersonAdresseResponse(
+                        navn = person.fullName ?: "",
+                        bostedsadresse = person.getBostedsadresse,
+                        kontaktadresse = person.getKontaktadresse,
+                        oppholdsadresse = person.getOppholdsadresse
+                    )
+                    call.respond(response)
+                } ?: call.respond(HttpStatusCode.NotFound)
```

Comment:
Vi må diskutere litt om det kanskje er bedre med NoContent.

Reply:
Vi gikk for `HttpStatusCode.InternalServerError` ettersom vi ikke vet om personen ikke finnes, eller om pdl er nede. Så vi gikk for det som fanger begge tilfeller.
Føltes ikke helt riktig å repsondere med en 2** kode hvis vi for eksempel får 500 fra pdl 😊